### ISSUE TITLE
Can I train an Italian model?

### ISSUE THREAD

**Issue by Pnlvfx** [Plachtaa/VITS-fast-fine-tuning#403]
Sorry but from the documentation I've not clearly understood it

**Comment by anfogy** [comment]
Yes you can, VITS take IPA instead of characters, if you wrote a framework on your own(take italian characters and turn them into IPA), you can do it.

**Comment by Molopo1114** [comment]
Can you give more detail for how to wrote a framework and apply in this VITS-fast fine turning ? Pleaseeeeeee
